src/layouts/MainLayout.astro

```diff
@@ -233,10 +231,9 @@ const updatedSince = timeSince(
           headings={headings}
           githubEditUrl={githubEditUrl}
         >
-          <p class='text-theme-muted'>
-            Last updated <span id="update-time" data-timestamp={lastModified}>{updatedSince}</span> ago <span
-              class='text-sm text-theme-muted/30 italic'
```

Comment:
Initial value was incorrect, because it was generating a timestamp of the server build time, instead of current date